libcontainer/standard_init_linux.go

```diff
@@ -109,6 +113,11 @@ func (l *linuxStandardInit) Init() error {
 	if err := syncParentReady(l.pipe); err != nil {
 		return err
 	}
+	if l.config.Config.NoNewPrivileges {
```

Comment:
If there's no reason to place it after `syncParentReady` (like `syncParentReady` might break it), I'd prefer to place it before. Go likes to create new processes each time you do a syscall, which is the whole point of late cgroup apply (applying cgroups as late as possible means we're less likely to hit our resource limits during setup).